Dockerfile

```diff
@@ -1,17 +1,8 @@
-from ubuntu:latest
+from gerbil/gambit
 
 MAINTAINER gerbil@cons.io
 
 ENV GAMBIT_VERSION v4.9.3
-RUN apt update -y
-RUN apt install -y libsqlite3-dev build-essential git autoconf libsnappy1v5 libleveldb1v5 zlib1g-dev libssl-dev pkg-config  libyaml-dev libmysqlclient-dev liblmdb-dev libleveldb-dev rsync texinfo
```

Comment:
we still need this stuff right? Or is it installed at the base layer?

Reply:
we move the gambit stuff out into the gerbil/gambit image, which we inherit from. This will make it so the gerbil build phase is all that is run from dockerhub on commits/prs.